### ISSUE TITLE
Battery plugin have action triggered by threshold performed only once

### ISSUE THREAD

**Issue by grenmester** [jaor/xmobar#515]
The battery monitor plugin currently has the `-a` and `-A` flags which allow you to perform an action if the battery level goes below a specified threshold. I would like to use this to send a low battery notification once the battery level goes below a certain threshold. I would like to do something like

```
Run Battery
  [ "--"
  , "-a", "notify-send 'Low Battery'"
  ] 50,
```

but the command following the `-a` flag is run every time the refresh rate is triggered. Is it possible to introduce an option where once a threshold is reached, an action is triggered but only once, i.e. the action is triggered when the battery level goes from a value above the threshold to a value below the threshold?

I understand that I could write a custom script to manually check this myself (https://www.reddit.com/r/bspwm/comments/chqxst/how_do_i_get_a_low_battery_notification/) but I was wondering if it would be more elegant to be able to use something builtin.

**Comment by jaor** [comment]
i agree it'd be a nice thing to have, even though, as you mention, it's
easy to work around it via scripts.
